flink-sql-connector-oracle-cdc/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,6 @@
+flink-sql-connector-oracle-cdc
+Copyright 2020 Ververica Inc.
+
+This project bundles the following dependencies under the Apache Software License 2.0. (http://www.apache.org/licenses/LICENSE-2.0.txt)
+
+- org.apache.kafka:kafka-clients:2.5.0
```

Comment:
Wait this seems wrong @mbalassi . When I grep for Kafka in this PR, I also get plenty of hits that are probably C&P errors. Could you please clean that up?

Reply:
CC @leonardBang

Reply:
Hi, @AHeise Sorry but I didn't get meaning about `C&P errors`, could you explain more ?
All sql connector jar are bundled just like we did in Flink community.

Reply:
c&p = copy and paste.
I don't think oracle cdc bundles kafka clients.

Reply:
We do bundle it, you can see `org.apache.kafka.common.utils.ThreadUtils`, but we can remove it as an  improvements.

The comment is valid although we doesn't depend on the Kafka cluster, the Embedded Debezium engine requires it.
```
[INFO] com.ververica:flink-sql-connector-oracle-cdc:jar:2.1-vvr-4.0.11-1-SNAPSHOT
[INFO] +- com.ververica:flink-connector-oracle-cdc:jar:2.1-vvr-4.0.11-1-SNAPSHOT:compile
[INFO] |  +- com.ververica:flink-connector-debezium:jar:2.1-vvr-4.0.11-1-SNAPSHOT:compile
[INFO] |  |  +- io.debezium:debezium-api:jar:1.5.2.Final:compile
[INFO] |  |  \- io.debezium:debezium-embedded:jar:1.5.2.Final:compile
[INFO] |  |     +- org.apache.kafka:connect-api:jar:2.7.0:compile
[INFO] |  |     |  +- org.apache.kafka:kafka-clients:jar:2.7.0:compile
```

Reply:
Okay, thank you for clarifying. You can ignore my comment then ;)